circle.yml

```diff
@@ -1,4 +1,17 @@
 machine:
   node:
     version: 4.3.0
-
+test:
+  override:
+    - ./node_modules/.bin/eslint --quiet:
```

Comment:
>maybe move to a `npm run lint` which runs eslint and move the makefile to use that?

I'm a bit of the opinion this is the direction we should head anyways :smile: (Related: #3520)

Is `npm` any better about passing arguments to the scripted commands? In my own experiences, I seem to recall this being an issue.